docs/user-documentation/metadata.md

```diff
@@ -1,50 +1,180 @@
 
-### Custom field: EDTF Date
+![Screenshot of the "Manage fields" page for the "Repository Item" content type from islandora_demo.](../assets/metadata_content_type_screenshot.png)
 
-### Custom field: Typed Relation
+> !!! tip: The included title field is limited to 255 characters; if your content has longer titles it is encouraged to create a separate long_title field to store the full title and reserve the default title field for a display title.
 
+> !!! tip "7.x Migration Note: What about my XML?"
+>
+> In 7.x, metadata were (usually) stored within XML datastreams such as MODS or DC. In CLAW we are breaking out
+individual metadata elements into fields instead of using an attached XML document. The Metadata Interest Group is developing a default mapping which will provide a basic, yet customizable, transform between MODS metadata and Drupal fields in Islandora Demo.
+>
+> It is still possible to attach an XML file to a CLAW object as a Media (see Datastreams), however there is no mechanism in CLAW for editing XML in a user-friendly way.
 
-## Mappings
+A specific instance of a content type is called a _node_, each of which has its own node identifier. In other words, a _node_ is the descriptive metadata for a particular digital object and the _content type_ is the node's metadata profile. Once a node is created, it cannot change its content type. To change a digital object's metadata profile (content type) a repository manager would need to create a new descriptive record (node) using the new metadata profile and then update the corresponding media records to point to the new descriptive record.
 
-Fields get mapped to RDF in (a yml file). See the RDF and Fedora section.
+## Vocabularies
 
-## Islandora demo
+In Drupal, _Taxonomy Vocabularies_ (or simply 'Vocabularies') are also entity subtypes that group fields and their configurations. Unlike content types, they are intended to be used as descriptive attributes of content and have hierarchy built in. Whereas instances of content types are called nodes, items in a vocabulary are called _terms_.
 
-TODO: document the fields and display modes in Islandora Demo.
+For example, Islandora includes the 'Islandora Models' vocabulary which includes the 'External URI' field. The 'Islandora Models' vocabulary includes the terms 'Audio', 'Binary', 'Collection', 'Image', and 'Video'. By linking to one of these terms in the 'Islandora Models' vocabulary a repository manager can tell that the node (digital object) should be considered an 'Image' or 'Audio', etcetera. The Controlled Access Terms module provides additional vocabularies representing Corporate Bodies, Persons, Families, Geographic Locations, and Subjects. Each of these vocabularies has its own set of fields allowing repositories to further describe them. Repository item nodes can then reference terms in these vocabularies. See 'Entity Reference fields' in the 'Field Types' section below.
```

Comment:
Another edit for brevity:

For example, Islandora includes the 'Islandora Models' vocabulary, which contains the terms 'Audio', 'Binary', 'Collection', 'Image', and 'Video'.